flow-osgi/pom.xml

```diff
@@ -69,6 +69,18 @@
             <artifactId>license-checker</artifactId>
             <version>1.1.1</version>
         </dependency>
+        
+         <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.13.1</version>
```

Comment:
Isn't here the scope missing? For mockito as well?

Reply:
Too late :(
I will make another PR.

Reply:
Excellent catch! I overlooked it, thanks!